src/renderer/components/+preferences/helm-charts.tsx

```diff
@@ -116,20 +115,18 @@ export class HelmCharts extends React.Component {
         <AddHelmRepoDialog onAddRepo={() => this.loadRepos()}/>
         <div className="repos flex gaps column">
           {Array.from(this.addedRepos).map(([name, repo]) => {
-            const tooltipId = `message-${name}`;
-
             return (
-              <Badge key={name} className="added-repo flex gaps align-center justify-space-between">
-                <span id={tooltipId} className="repo">{name}</span>
+              <div key={name} className="repo flex gaps align-center justify-space-between">
+                <div>
+                  <div className="repoName">{name}</div>
+                  <div className="repoUrl">{repo.url}</div>
```

Comment:
Are you sure that the URL won't be too long in general?

Reply:
They will wrap text content onto new lines, so nothing bad happened in this case.